feature-libs/order/components/order-history/order-history.component.ts

```diff
@@ -29,11 +35,13 @@ export class OrderHistoryComponent implements OnDestroy {
     protected routing: RoutingService,
     protected orderHistoryFacade: OrderHistoryFacade,
     protected translation: TranslationService,
-    protected replenishmentOrderHistoryFacade: ReplenishmentOrderHistoryFacade
+    protected replenishmentOrderHistoryFacade: ReplenishmentOrderHistoryFacade,
+    @Optional() protected featureConfigService?: FeatureConfigService
   ) {}
 
   private PAGE_SIZE = 5;
   sortType: string;
+  hasPONumber = false;
```

Comment:
Could we use an observable for this so we can use `async` pipe in the template?

Reply:
We don't want to construct multiple static observables and re-assign the observable variable, the same applies to the existing 'sortType'

Reply:
Okay. Lets leave this undefined like `hasPONumber: boolean` as the true/false value will become meaningful when it is instantiated later.

Reply:
done thanks~